### PR TITLE
netstat: use long argument format

### LINKED ISSUE #5251: netstat: duplicate example for routing table
It seems that the pages for `netstat` have a duplicated example for printing the routing table, with the order of the flags switched and a slightly different description.

```
  - List routes and do not resolve IP to hostname:
    netstat -rn
```

vs

```
  - Print the routing table:
    netstat -nr
```

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Close #5251

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@navarroaxel will you please do this for the macOS and macOS-Chinese versions?

- [`pages/osx/netstat.md`](https://github.com/tldr-pages/tldr/blob/master/pages/osx/netstat.md)
- [`pages.zh/osx/netstat.md`](https://github.com/tldr-pages/tldr/blob/master/pages.zh/osx/netstat.md)

**Comment by navarroaxel** [comment]
@bl-ue Looks like the macOS version doesn't have long arguments, but I added the more information link. This is the help:
```
Usage:  netstat [-AaLlnW] [-f address_family | -p protocol]
        netstat [-gilns] [-f address_family]
        netstat -i | -I interface [-w wait] [-abdgRtS]
        netstat -s [-s] [-f address_family | -p protocol] [-w wait]
        netstat -i | -I interface -s [-f address_family | -p protocol]
        netstat -m [-m]
        netstat -r [-Aaln] [-f address_family]
        netstat -rs [-s]
```

**Comment by bl-ue** [comment]
Ah ok, I could've checked that myself :smile:

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
Note to maintainers: please **rebase** this PR as the commit history is clean.

File: pages/linux/netstat.md [hunk 1]
```diff
@@ -1,35 +1,32 @@
...
 
-`netstat -p`
+`netstat --program`
 
 - List information continuously:
 
-`netstat -c`
+`netstat --continuous`
 
 - List routes and do not resolve IP to hostname:
```
Comment (sbrl) [inline pages/linux/netstat.md:26]: Perhaps this will read better?
```suggestion
- List routes and do not resolve IP addresses to hostnames:
```

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
>Note to maintainers: please rebase this PR as the commit history is clean.

This is no longer true with 48dcef0a1ea16c6d212b5cd9c39acd827ae5523a, shall I obsolete that comment or are you going to force-push to clean up?

**Review by sbrl — APPROVED** [review]
Awesome, @navarroaxel!

**Pull request merged**